### ISSUE TITLE
Raw <p> tag is shown on radio group

### ISSUE THREAD

**Issue by RyanRussell00** [calcom/cal.com#12028]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

When a form to submit for booking has a radio group, the label is prefixed with <p>

### Steps to Reproduce

1. Create a radio group in your event type
2. Set a label
3. View it in your calendar as if you're trying to book a time and noticed the raw <p>

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Actual Results

Shows <p>

### Expected Results

Should not be showing raw html

### Technical details

- Chrome on MacBook
- Date: Oct 21, 2023 around 4:50pm PST

### Evidence

<img width="477" alt="image" src="https://github.com/calcom/cal.com/assets/35180491/fb9b9917-dbdf-4818-a587-b5a5a7753996">

**Comment by sikehish** [comment]
Hey, I'm looking into it.

**Comment by DarshanBorse** [comment]
I'm ready to work on the issue; please assign it to me.

**Comment by codesmith922** [comment]
@RyanRussell00 
Bug steps are not mentioned correctly. Where is this checkbox? Please add more steps.

**Comment by itskish0re** [comment]
@RyanRussell00 may be you used <p> tag in label input.
something like this 
![image](https://github.com/calcom/cal.com/assets/59221338/070fd4e7-0bd8-48c3-9c14-c074e4ac1793)

**Comment by Abhinav-Developer-23** [comment]
I am not able to replicate this

**Comment by itskish0re** [comment]
@Abhinav-Developer-23 check this issue #12030

**Comment by PeerRich** [comment]
cannot reproduce.

TLDR: do not use html tags inside as names

![CleanShot 2023-10-23 at 12 18 46@2x](https://github.com/calcom/cal.com/assets/8019099/69be71b7-44e4-46a6-b842-87ce153264e8)
